zulipterminal/ui_tools/boxes.py

```diff
@@ -695,12 +695,19 @@ def footlinks_view(
                              align='left', left=8, width=('relative', 100),
                              min_width=10, right=2)
 
-    def soup2markup(self, soup: Any, **state: Any) -> List[Any]:
+    @classmethod
+    def soup2markup(cls, soup: Any, server_url: str, metadata: Dict[str, Any],
+                    message_links: 'OrderedDict[str, Tuple[str, int, bool]]',
```

Comment:
Is this quoting since `OrderedDict` doesn't have a `typing` version?

Reply:
Yes, mypy doesn't work well with `OrderedDict`. We have a comment for it in `footlinks_view` as well.